nuxeo-features/nuxeo-elasticsearch/nuxeo-elasticsearch-core/src/main/java/org/nuxeo/elasticsearch/core/ElasticSearchIndexingImpl.java

```diff
@@ -178,6 +180,12 @@ void processBulkIndexCommands(List<IndexingCommand> cmds) {
                     bulkSize += idxRequest.source().length();
                     bulkRequest.add(idxRequest);
                 }
+            } catch (PropertyException pe) {
+                if (pe.getMessage().contains(CANNOT_GET_BLOB_INFO)) {
```

Comment:
This is very brittle, at the very least add a message where `CANNOT_GET_BLOB_INFO` is defined to explain what produces this message. But more generally consider throwing a specific subexception and catching it here.

Reply:
I discussed this at the time.  I didn't want to change behaviour in a backport by throwing a new exception but subexception would work.